packages/card-support/package.json

```diff
@@ -56,6 +61,5 @@
     "i18next-parser": "^3.5.0"
   },
   "author": "OpenSRP Engineering",
-  "license": "Apache-2.0",
-  "private": false
```

Comment:
why? I think lerna requires this to be false in order to publish

Reply:
It was automatically removed by one of the yarn versions I tried installing, left it because I still intend to migrate to the newer yarn versions. No lerna does not require this to be false in order to publish, it however will not publish if it is set to true.